package.json

```diff
@@ -49,7 +49,7 @@
   ],
   "dependencies": {
     "domhandler": "5.0.3",
-    "html-dom-parser": "5.0.10",
+    "html-dom-parser": "5.0.11",
```

Comment:
Don't forget to update `package-lock.json` as well

Reply:
@kingflamez the `html-dom-parser` package has been released:

- https://github.com/remarkablemark/html-dom-parser/pull/910#issuecomment-2515876575

Reply:
Heads up, I'll be merging #1623 so we can close this PR. Thanks for your help on this!